pkg/ekcoops/operator.go

```diff
@@ -231,31 +232,37 @@ func (o *Operator) adjustPoolReplicationLevels(rookVersion semver.Version, numNo
 
 	var multiErr error
 
+	cephVersion := semver.Version{}
 	cephcluster, err := o.controller.GetCephCluster(context.TODO())
 	if err != nil {
 		multiErr = multierror.Append(multiErr, errors.Wrapf(err, "get CephCluster config"))
+	} else {
+		cephVersion, err = rook.GetCephVersion(*cephcluster)
+		if err != nil {
+			multiErr = multierror.Append(multiErr, errors.Wrapf(err, "get ceph version"))
+		}
 	}
 
-	didUpdate, err := o.controller.SetBlockPoolReplication(rookVersion, o.config.CephBlockPool, factor, cephcluster, doFullReconcile)
+	didUpdate, err := o.controller.SetBlockPoolReplication(rookVersion, cephVersion, o.config.CephBlockPool, factor, doFullReconcile)
```

Comment:
just to make sure, these all work fine with an empty cephVersion object?

Reply:
i made it more explicit